src/resources/postcss/utilities/variables/_svgs.pcss

```diff
@@ -1,10 +1,10 @@
 /* -----------------------------------------------------------------------------
- * SVG - Icon Featured
+ * SVG - Featured
  * ----------------------------------------------------------------------------- */
 
-@svg-load icon-featured url('../../../images/icons/featured-event.svg') {
+@svg-load featured url('../../../images/icons/featured-event.svg') {
```

Comment:
Might be helpful to have the path vars like we have on SQ1, not a blocker or requirement.